### ISSUE TITLE
$unset directive does not process arrays

### ISSUE THREAD

**Issue by jumale** [kolodny/immutability-helper#44]
$unset directive expects an array, but processes only one element for the array, other elements are ignored.

Here is an example:
```javascript
let state = {
  foo: true,
  bar: true,
  baz: true
};
let newState = update(state, {$unset: ['foo', 'bar']});
console.log(newState);
// prints {foo: true, baz: true}
// expected {baz: true}
```

**Comment by kolodny** [comment]
Thanks for reporting this issue. There's currently a PR to address this #43

**Comment by kolodny** [comment]
This should be fixed in 7285de4 and released now. Please double check to make sure that the issue is fixed. Thanks
